app/Jobs/CheckQueueIssues.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Jobs;
+
+use App\Mail\QueueErrorMail;
+use App\SuperUser;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Support\Facades\DB;
+
+class CheckQueueIssues implements ShouldQueue
+{
+    use Dispatchable, InteractsWithQueue, Queueable, SerializesModels;
+
+    /**
+     * Execute the job.
+     */
+    public function handle()
+    {
+        if (DB::table('jobs')->count() > 9) {
```

Comment:
Make it send only if the jobs haven't been executed for an hour. (we'll need to test this).
```suggestion
        if (DB::table('jobs')->where('updated_at', '<', now()->subHour()),->count() > 9) {
```